tools/cli/build.gradle

```diff
@@ -33,17 +30,26 @@ task dumpOSInfo {
     }
 }
 
-task copyCLIShortcut(type: Copy, dependsOn: [distBinary, dumpOSInfo]) {
-    String go_osname     = mapOsNameToGoName(getOsName())
-    String go_osarch     = mapOsArchToGoArch(getOsArch())
-    String from_path_wsk = "${projectDir}/../../bin/${go_osname}/${go_osarch}/wsk"
-    String to_path_dir   = "${projectDir}/../../bin"
-
-    from from_path_wsk
-    into to_path_dir
+task copyCLIShortcut() {
+    doLast {
+        String go_osname = mapOsNameToGoName(getOsName())
+        String go_osarch = mapOsArchToGoArch(getOsArch())
+        String from_path_wsk = "${projectDir}/../../bin/${go_osname}/${go_osarch}/wsk"
+        String to_path_dir = "${projectDir}/../../bin"
+
+        if (file(from_path_wsk).exists()) {
+            exec {
+                executable "cp"
+                args "${from_path_wsk}", "${to_path_dir}"
+            }
+        }
+    }
```

Comment:
If I recall correctly the exec functionallity worked better on the Windows machine running Vagrant.